### PR TITLE
Permissions related information added

### PR DESCRIPTION
**Pull request by joinimran**
As requested by user, Added permissions information needed to create, test, and deploy a custom sensitive information type through the UI?

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/820

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/create-a-custom-sensitive-information-type.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ You can also create custom sensitive information types by using PowerShell and E
 
 ## Before you begin...
 
+> [!Note]
+> You should have global admin or compliance admin permissions granted to create, test, and deploy a custom sensitive information type through the UI. See [About admin roles](https://docs.microsoft.com/office365/admin/add-users/about-admin-roles?view=o365-worldwide) in Office 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-custom-sensitive-information-type.md]:
```suggestion
> You should have Global admin or Compliance admin permissions granted to create, test, and deploy a custom sensitive information type through the UI. See [About admin roles](https://docs.microsoft.com/office365/admin/add-users/about-admin-roles?view=o365-worldwide) in Office 365.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy Review - minor edits

**Comment by joinimran** [comment]
Suggestions applied. Ready to go live @mypil

**Pull request merged**